hash/msx2_flop.xml

```diff
@@ -11203,38 +13564,51 @@ The following floppies came with the machines.
 		<description>The Tower? of Cabin - Cabin Panic (Japan)</description>
 		<year>1992</year>
 		<publisher>Micro Cabin</publisher>
+		<info name="alt_title" value="THE TOWER OF CABIN?～キャビンパニック～"/>
```

Comment:
Is the question mark supposed to be in a different place in the description and `alt_title`?

Reply:
![the_tower_of_cabin](https://user-images.githubusercontent.com/9251271/223259594-86368a54-4a85-4eaf-937e-f8daee16d7bf.png)